test/e2e/create_test.go

```diff
@@ -222,6 +223,53 @@ func (tc *testContext) waitForMachinesDeleted(machines []mapi.Machine) (err erro
 	return err
 }
 
+// machineAwsEC2LogCollection collects specific system logs from the AWS EC2 instances
+func (tc *testContext) machineAwsEC2LogCollection(machines []mapi.Machine) {
+	if tc.CloudProvider.GetType() != config.AWSPlatformType {
+		log.Printf("Skipping AWS EC2 log collection for %s", tc.CloudProvider.GetType())
+		return
+	}
+	for _, machine := range machines {
+		address, err := controllers.GetAddress(machine.Status.Addresses)
+		if err != nil {
+			log.Printf("Machine %s does not have a valid address, unable to get logs", machine.GetName())
+			continue
+		}
+		tc.collectAwsEC2UserDataLogs(address)
+		// TODO: Collect EC2 Launch Agent logs for the given address. e.g. tc.collectAwsEC2LaunchAgentLogs(address)
+		// see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2launchv2-troubleshooting.html#ec2launchv2-console-log-output
```

Comment:
Why should this be done?
If it needs to be done a ticket should be created

Reply:
This is nice to have, I'd say. It may help to triage any EC2Launch issues; as of now, in CI, we are in the dark about EC2Launch logs. 

Happy to file-in a ticket or drop the TODO, I will defer to you.

Reply:
Personally I would say drop the TODO, as it will realistically not get done based on our track record :) 
But I am fine either way.

Reply:
Dropped the TODO in https://github.com/openshift/windows-machine-config-operator/pull/2296/commits/5b16c64a87fee43de0bd5607ff8a6e846e303c4c